installation/language/de-DE/de-DE.ini

```diff
@@ -208,76 +185,46 @@ INSTL_DATABASE_INVALID_MYSQLI_VERSION="Um die Installation fortzusetzen wird MyS
 
-;Controllers
+;controllers
 INSTL_COOKIES_NOT_ENABLED="Ihr Browser scheint keine Cookies zu unterstützen!<br />Die Anwendung kann nicht installiert werden, wenn diese Browserfunktion deaktiviert ist! Alternativ könnte auch ein Problem mit dem <strong>session.save_path</strong> des Servers vorliegen. Wenn dies der Fall ist, ist der Webhoster zu kontaktieren, falls das Problem nicht anders behoben oder festgestellt werden kann."
 INSTL_HEADER_ERROR="Fehler"
 
 ;Helpers
 INSTL_PAGE_TITLE="Joomla!-Webinstallation"
 
 ;Configuration model
-INSTL_ERROR_CONNECT_DB="Es konnte keine Verbindung zur Datenbank hergestellt werden! Der Konnektor gab folgenden Fehler zurück: %s"
-INSTL_STD_OFFLINE_MSG="Diese Website ist zurzeit im Wartungsmodus.<br />Bitte später wiederkommen."
-
-;FTP model
-INSTL_FTP_INVALIDROOT="Das angegebene FTP-Verzeichnis entspricht nicht dem Verzeichnis dieser Joomla!-Installation!"
-INSTL_FTP_NOCONNECT="Es konnte keine Verbindung mit dem FTP-Server hergestellt werden!"
-INSTL_FTP_NODELE="Die Funktion „DELE“ schlug fehl!"
-INSTL_FTP_NODIRECTORYLISTING="Es konnte keine Verzeichnisauflistung vom FTP-Server erstellt werden!"
-INSTL_FTP_NOLIST="Die Funktion „LIST“ schlug fehl!"
-INSTL_FTP_NOLOGIN="Anmeldung am FTP-Server nicht möglich!"
-INSTL_FTP_NOMKD="Die Funktion „MKD“ schlug fehl!"
-INSTL_FTP_NONLST="Die Funktion „NLST“ schlug fehl!"
-INSTL_FTP_NOPWD="Die Funktion „PWD“ schlug fehl!"
-INSTL_FTP_NORETR="Die Funktion „RETR“ schlug fehl!"
-INSTL_FTP_NORMD="Die Funktion „RMD“ schlug fehl!"
-INSTL_FTP_NOROOT="Auf das angegebene FTP-Verzeichnis kann nicht zugegriffen werden!"
-INSTL_FTP_NOSTOR="Die Funktion „STOR“ schlug fehl!"
-INSTL_FTP_NOSYST="Die Funktion „SYST“ schlug fehl!"
-INSTL_FTP_UNABLE_DETECT_ROOT_FOLDER="Es war nicht möglich das FTP-Root-Verzeichnis zu ermitteln!"
+INSTL_ERROR_CONNECT_DB="Die Verbindung zur Datenbank konnte nicht hergestellt werden. Es kam folgende Fehlermeldung zurück: %s"
+INSTL_STD_OFFLINE_MSG="Diese Seite ist wegen Wartungsarbeiten nicht erreichbar.<br>Bitte schauen Sie bald wieder vorbei."
 
-;Others
-INSTL_CONFPROBLEM="Entweder ist die Konfigurationsdatei oder das Verzeichnis schreibgeschützt oder es gab ein anderes Problem beim Speichern der Konfiguration. Der folgende PHP-Code muss daher manuell hochgeladen werden. Bitte den gesamten Text im nachfolgenden Textfeld markieren, dann in eine neue Datei mit dem Namen „configuration.php“ kopieren und diese dann in das Hauptverzeichnis der Joomla!-Website hochladen."
-INSTL_DATABASE_SUPPORT="Datenbankunterstützung:"
+;others
+INSTL_CONFPROBLEM="Ihre Konfigurationsdatei oder Ihr Ordner ist nicht beschreibbar oder es gab ein Problem bei der Erstellung der Konfigurationsdatei. Sie müssen den folgenden Code von Hand hochladen. Wählen Sie im Textfeld, um den gesamten Code zu markieren, und fügen Sie ihn dann in eine neue Textdatei ein. Benennen Sie diese Datei'configuration.php' und laden Sie sie in Ihren Stammordner hoch."
```

Comment:
INSTL_CONFPROBLEM="Die Konfigurationsdatei oder der Ordner ist nicht beschreibbar oder es gab ein Problem bei der Erstellung der Konfigurationsdatei. Der folgende Code muss im Anschluß von Hand hochgeladen werden. Ein Klick in das Textfeld wird den gesamten Code markieren, anschließend ist der Code in eine neue Textdatei einzufügen. Diese Datei muss den Namen 'configuration.php' haben und in das Joomla!-Root-Verzeichnis hochgeladen werden."

Reply:
In Anlehnung an die alte Version aus 3.9.5